orgs/beta.json

```diff
@@ -1,12 +1,16 @@
 {
-  "edition": "Developer", 
   "orgName": "V4S - Beta Test Org", 
+  "edition": "Developer", 
+  "hasSampleData": "true",
   "features": "Sites",  
   "orgPreferences": {
     "enabled": [
       "ChatterEnabled", 
       "S1DesktopEnabled", 
       "DisableParallelApexTesting"
+    ],
+    "disabled": [
+      "S1EncryptedStoragePref2"
```

Comment:
@jjbennett I think we should leave the lightning cache enabled for org shapes that are used for QA & testing (feature, beta, release) so that they are more similar to production orgs. It makes sense to turn it off for dev orgs.

Reply:
I think it also makes sense for QA otherwise you have to refresh a ton to see updated data / layout changes.